### PR TITLE
[MODGOBI-178] Logging improvement - Configuration

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose

https://issues.folio.org/browse/MODGOBI-178
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [ ] Duplications on new code is 3% or less
  - [ ] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by azizbekxm** [comment]
image

**Comment by azizbekxm** [comment]
image

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -9,6 +9,7 @@
 -->
 
 ## Purpose
+
```
Comment (Abdulkhakimov) [inline PULL_REQUEST_TEMPLATE.md:12]: Please rollback these changes and leave only the line which you added the step to check logging. Thank you

File: src/test/resources/log4j2.properties [hunk 2]
```diff
@@ -1,18 +1,33 @@
...
+appender.console.layout.userId.key = userId
+appender.console.layout.userId.value = $${FolioLoggingContext:userid}
+
+appender.console.layout.moduleId.type = KeyValuePair
+appender.console.layout.moduleId.key = moduleId
+appender.console.layout.moduleId.value = $${FolioLoggingContext:moduleid}
 
 rootLogger.level = debug
-rootLogger.appenderRefs = info
+rootLogger.appenderRefs = debug
```
Comment (Abdulkhakimov) [inline src/test/resources/log4j2.properties:28]: I don't recommend to decrease logging level

File: src/test/resources/log4j2.properties [hunk 3]
```diff
@@ -1,18 +1,33 @@
...
 rootLogger.appenderRef.stdout.ref = STDOUT
+
+logger.netty.name = io.netty
```
Comment (Abdulkhakimov) [inline src/test/resources/log4j2.properties:31]: Why do you add netty properties?

Comment (azizbekxm) [reply src/test/resources/log4j2.properties]: It is like misunderstanding. I have removed them. Thanks for CR.

**Review by okolawole-ebsco — APPROVED** [review]

**Pull request merged**